content/en/docs/setup/getting-started/index.md

```diff
@@ -348,6 +348,30 @@ by viewing the Bookinfo product page using a browser.
 
 1.  Paste the output from the previous command into your web browser and confirm that the Bookinfo product page is displayed.
 
+### Apply default destination rules
```

Comment:
One question I have, and I know @craigbox talked about moving this section up (I prefer to keep it the example back and here), is if we want to outdent this section like the BookInfo page and use ##?

Reply:
Thanks.